libass/ass_blur.c

```diff
@@ -744,7 +744,7 @@ static void calc_coeff(double mu[4], const int index[4], int prefilter, double r
         (  17 -  126 * mul +  273 * mul2 -  164 * mul3) / 12096,
     };
 
-    double mat_freq[13];
+    double mat_freq[14];
```

Comment:
Wow, this function is so extremely hard to follow. Is it right that vec_freq below still has only 13 items?